hack/presubmit.sh

```diff
@@ -37,6 +37,27 @@ warn() {
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 CRD_DIR=${REPO_ROOT}/charts/operator/crds
 
+codegen_diff() {
```

Comment:
nit: do you know when this was removed? I'm surprised it didn't break anything when it happened...